src/chromo/models/pythia8.py

```diff
@@ -71,7 +71,7 @@ class Pythia8(MCRun):
         + "/releases/download/zipped_data_v1.0/Pythia8_v002.zip"
     )
 
-    def __init__(self, evt_kin, *, seed=None, config=None):
+    def __init__(self, evt_kin=None, *, seed=None, config=None, banner=True):
```

Comment:
evt_kin should not have a default value of none, please change that back. Noramlly, you cannot initialize a generator without initial state. The situation for Pythia8 is special, but that does not justify breaking this general expectation. You can still pass None for evt_kin even if it is not the default value.

This function signature was specifically designed with the * so that the only accepted positional argument is evt_kin.

Reply:
Default None is removed.